### DISCUSSION TITLE
Flag "Dicitura Sconto a voi riservato" tramite API

### DISCUSSION THREAD

**Discussion by tobiascapin** [fattureincloud/api#471] [Q&A] [answered]
Buongiorno,
sto creando delle fatture tramite API, trovo il campo per inserire lo sconto %, ma non trovo il flag per definire l'opzione nella riga articolo "Dicitura Sconto a voi riservato". Esiste o manca? Grazie

**Comment by emajo** [comment, marked as answer]
Ciao @tobiascapin il campo che sta cercando è `discount_highlight` specificabile all'interno degli items.

Grazie,
Emanuele.

**Comment by tobiascapin** [comment]
Grazie mille!
